_common.R

```diff
@@ -100,6 +100,7 @@ assays_to_be_censored_at_uloq_cor <- c(
 # figure labels and titles for markers
 ###############################################################################
 has29 = "Day29" %in% times
+has57 = study_name_code=="COVE"
```

Comment:
Why not `"Day57" %in% times`? Might generalize better when we need to handle AZ/Nova.